src/Components/Header.js

```diff
@@ -0,0 +1,18 @@
+// Auto generate rafce(Creates an arrow functio). (By using the ES7 extension)
+const Header = ({ name }) => {
+  return (
+    <ul>
+      <li>
+        <header className="welcome">
+          <h2>
+            Welcome, &nbsp;<span>{name} </span>
+          </h2>
+        </header>
+      </li>
+    </ul>
```

Comment:
Hmm. Again with the list. Don't use lists unless you actually have a list of elements